src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -293,6 +294,38 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
             return matchingVersion?.ToString();
         }
 
+        private NuGetVersion ResolvePlatformReleaseChannelVersion(IEnumerable<NuGetVersion> bundleVersions, NuGetVersion currentMatchingVersion)
+        {
+            bool isStandardOrExtendedChannel = string.Equals(_platformReleaseChannel, ScriptConstants.StandardPlatformChannelName, StringComparison.Ordinal) ||
```

Comment:
> Extended: Resolves to the version two releases prior to the latest (n-2), if that version is available.
However, Functions and Rapid Update should treat Standard and Extended the same, resolving to n-1.

If we are intentionally returning n-1 for both cases, we should add a comment explaining "Why" we are doing what we are doing, for future readers.